docs/guides/cdk/creating-exports-file.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Creating a CDK exports file
+description: Creating the exports file for consumption by Amplify
+---
+
+When using Amplify client libraries, you can consume resources from a CDK backend by creating a CDK exports file and then configuring Amplify to consume it.
+
+### Creating the exports file
+
+Using the CDK `CfnOutput` command, you can print out and the write to file the values from your CDK app that Amplify client libraries require.
+
+For example, a CDK app that implements an AppSync API with multiple authorization types and a Cognito User Pool could use the following to print out the values:
+
+```ts
+new cdk.CfnOutput(this, "aws_user_pools_id", {
```

Comment:
bear in mind that the actual logical ID of the `CfnOutput` depends on the scope in which the output is defined. If it is defined directly under a `Stack` the construct ID will be the same ass the logical ID. If it's defined within some nested construct, the logical ID will be mangled.

If those IDs are important (and I assume they are), then I would mention that those outputs must be defined directly within the `Stack`'s scope.

Reply:
Got it, this is good insight. So if this was written in a different sack in the same project then you would see multiple stack names in the output?